bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingReadOp.java

```diff
@@ -78,6 +78,7 @@
     final boolean isRecoveryRead;
     boolean parallelRead = false;
     final AtomicBoolean complete = new AtomicBoolean(false);
+    boolean allowFastFail = false;
```

Comment:
`fail-fast` is better than `fast-fail`, since `fail-fast` is a well known term : https://en.wikipedia.org/wiki/Fail-fast